server/models/video/video.ts

```diff
@@ -740,9 +740,20 @@ export class VideoModel extends Model<VideoModel> {
     return this.remote === false
   }
 
-  createPreview (videoFile: VideoFileModel) {
-    const imageSize = PREVIEWS_SIZE.width + 'x' + PREVIEWS_SIZE.height
+  createPreview (videoFile: VideoFileModel, videoRatio: number) {
+    const previewRatio = PREVIEWS_SIZE.width / PREVIEWS_SIZE.height
+    let previewHeight: number = PREVIEWS_SIZE.height
+    let previewWidth: number = PREVIEWS_SIZE.width
+
+    if(previewRatio >= videoRatio){
```

Comment:
Please use an editor that support tslint so you don't bother with code format. This line should be 

```if (previewRatio >= videoRatio) {```